example/cordova/www/pages/metrics.html

```diff
@@ -45,20 +45,31 @@ <h4>Response:</h4>
   return {
     beforeCreate() {
       var self = this;
-      self.$("#metrics-info").html("No metrics");
-      window.MobileCore.MetricsService.getMetrics(
-        function (metrics) {
+      self.$("#metrics-info").html("Fetching metrics...");
+
+      self.metricsService = new aerogear.MetricsService({ url: "http://192.168.1.6:3000/metrics" });
+      self.metricsService.getAppAndDeviceMetrics()
+        .then(metrics => {
           self.$("#metrics-info").html(JSON.stringify(metrics));
```

Comment:
Haven't checked how it looks currently, but the json should be wrapped in `<pre><code>`

Reply:
It looks horrible hehe, this actually could or should be removed and let only sendAppAndDeviceMetrics